Tryouts/Core/Abstractions/README.md

```diff
@@ -0,0 +1,9 @@
+<!-- Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. -->
+
+# MorganStanley.ComposeUI.Tryouts.Core.Abstractions
+===================================================
```

Comment:
Nitpick: the '#' and the series of '==' characters are two alternative syntaxes for the same, you don't need to combine them. See the basic Markdown syntax here: https://www.markdownguide.org/basic-syntax/